src/runtime/runtime.cs

```diff
@@ -664,7 +664,8 @@ internal static Type[] PythonArgsToTypeArray(IntPtr arg, bool mangleObjects)
 
                 if (mt is ClassBase)
                 {
-                    t = ((ClassBase)mt).type;
+                    MaybeType _type = ((ClassBase)mt).type;
+                    t = _type.Valid ?  _type.Value : null;
```

Comment:
NIT: couldn't this be `.type.UnsafeValue`?

Reply:
`MaybeType` doesn't have an UnsafeValue, and it's the only place where this pattern is used for `MaybeType`.